substrafl_examples/get_started/run_mnist_torch.py

```diff
@@ -483,9 +480,9 @@ def __init__(self):
 plt.xlabel("Rounds")
 plt.ylabel("Accuracy")
 
-for id in DATA_PROVIDER_ORGS_ID:
-    df = performances_df.query(f"worker == '{id}'")
-    plt.plot(df["round_idx"], df["performance"], label=id)
+for i in DATA_PROVIDER_ORGS_ID:
+    df = performances_df.xs(i, level="worker")
```

Comment:
Not a strong opinion but `xs` might not ne known by most of the DS (I did not know what is was doing).

Reply:
I find the slicers option (`performances_df.loc[(slice(None), i), :]`)  even less readable, but it's very subjective. 

We can also drop usage of the `MultiIndex` if you think it's too complex.